test/acceptance/features/bindAppToServiceUsingSecret.feature

```diff
@@ -394,3 +394,78 @@ Feature: Bind values from a config map referred in backing service resource
         Then Service Binding "ssd-2" is ready
         And The application env var "BACKEND_USERNAME" has value "AzureDiamond"
         And The application env var "BACKEND_PASSWORD" has value "hunter2"
+
+    Scenario: Inject into app all keys from a secret existing in a namespace different from the service resource
+        Given The Custom Resource Definition is present
+            """
+            apiVersion: apiextensions.k8s.io/v1beta1
+            kind: CustomResourceDefinition
+            metadata:
+                name: backends.stable.example.com
+                annotations:
+                    service.binding: path={.status.credentials},objectType=Secret
+            spec:
+                group: stable.example.com
+                versions:
+                  - name: v1
+                    served: true
+                    storage: true
+                scope: Namespaced
+                names:
+                    plural: backends
+                    singular: backend
+                    kind: Backend
+                    shortNames:
+                      - bk
+            """
+        * Namespace is present
+            """
+            apiVersion: v1
+            kind: Namespace
+            metadata:
+                name: backend-services
+            """
+        * The Secret is present
+            """
+            apiVersion: v1
+            kind: Secret
+            metadata:
+                name: ssa-3-secret
+                namespace: backend-services
+            stringData:
+                username: AzureDiamond
+                password: hunter2
+            """
+        * The Custom Resource is present
+            """
+            apiVersion: stable.example.com/v1
+            kind: Backend
+            metadata:
+                name: ssa-3-service
+                namespace: backend-services
```

Comment:
I can see that service `ssa-3-service` lives in `backend-services` namespace
and secret `ssa-3-secret` lives in `backend-services` namespace.

Both service and secret are in the same namespace... but the title of the PR says that we want to verify a situation where service and secret live in different namespace.

Reply:
Sorry, @DhritiShikhar I'll update the title service and secret should be in same ns. This test is based on issue #646.